Tests/test_class.py

```diff
@@ -35,7 +33,9 @@ def test_common_attributes(self):
             # Modifying __class__ causes a TypeError
             self.assertRaises(TypeError, i.__setattr__, "__class__")
 
-            self.assertEqual(type(i), i.__getattribute__("__class__"))
+            if i != 0:
```

Comment:
I thought I saw some code handling `__class__` . Do we need to skip?

Reply:
NO! Good catch!

I have skipped this test because I had originally intended `(1).__class` to be `Int32` as a convenient way to check the actual type without importing `System`, but it turned out it has to be `int` because StdLib assumes it is always so for integers (see: `@functools.singledispatch`). I guess I forgot about this test afterwards.